### DISCUSSION TITLE
node下载图片到本地报错，提示【node:internal/modules/cjs/loader:1249   throw err】

### DISCUSSION THREAD

**Discussion by neowukong** [Chilfish/Weibo-archiver#62] [Q&A]
![Clip_2024-11-04_01-34-35](https://github.com/user-attachments/assets/ca7243af-d613-44e1-9364-54c373e41206)
一切均是按步骤操作

**Comment by Chilfish** [comment]
这个错误是说找不到 download.mjs 文件，但下载图片的脚本应该是放在 script.zip 压缩包里的，你需要完整地解压这个文件夹，而不是直接打开它。文件 `下载微博图片.bat` 和 `download.mjs` 是会放在一起的才对

![image](https://github.com/user-attachments/assets/63b7636b-13df-4832-a8c2-7701e29c3293)

**Reply by neowukong** [reply]
是放在一起的哦
![Clip_2024-11-04_20-11-09](https://github.com/user-attachments/assets/4b762d1b-1cbb-4add-b3c4-57d7770618f7)

**Reply by neowukong** [reply]
脑壳有点大呀，整了好久了

**Reply by Chilfish** [reply]
![image](https://github.com/user-attachments/assets/7f7650f2-0193-4b9d-9743-b9e552efa502)

可以这样，在这个文件夹的地址栏输入 cmd 然后回车，在弹出来的黑窗口再输入 `node download.mjs -i ` -i 后接上 csv 文件的路径，应该就可以直接下载了

**Comment by neowukong** [comment]
![Clip_2024-11-08_20-07-43](https://github.com/user-attachments/assets/2a1c5e48-264c-40f4-96ed-f00b07727c28)
还是不行呀

**Reply by Chilfish** [reply]
这应该是windows文件路径的问题……，把-i 后面那串地址换成 `"imgs-INGINGkids（2）.csv"` 试试，也就是只用那个csv的文件名，要包括英文双引号

**Comment by neowukong** [comment]
已经解决了，需要ba把csv文件名的数字和空格去掉
![Clip_2024-11-08_20-56-45](https://github.com/user-attachments/assets/50579d36-a147-4931-903c-fec66ad74d3d)

**Reply by Chilfish** [reply]
okok，之后改成自动找这个 csv 后缀的文件好了

**Reply by neowukong** [reply]
文件名的兼容性不是很好，之前下载了好久次，文件名有数字后，就识别不了了
